src/index.cjs

```diff
@@ -91,6 +97,12 @@ class ScratchWebpackConfigBuilder {
                     {
                         test: enableReact ? /\.[cm]?jsx?$/ : /\.[cm]?js$/,
                         loader: 'babel-loader',
+                        exclude: [
+                            {
+                                and: [/node_modules/],
+                                not: [/node_modules\/scratch-/]
```

Comment:
Just thinking out loud: Technically we have a few repos that don't fit this naming scheme, including `eslint-config-scratch` and `Paper.js`. We may not need to account for `eslint-config-scratch` because I don't expect it to be a webpack input, but `Paper.js` will be. On the other hand, it may be safe enough to consider semi-external, especially since it's a fork that won't be joining the monorepo.

Also: does this work on Windows, or does the `/` need to be something silly like `[\\/]`?

Reply:
The idea is `\` escapes `/`. You are right about the regex, I can change it to `node_modules\/.*scratch.*` and this way it will match scoped names etc. as well

Reply:
> The idea is \ escapes /.

Ah, sorry, I wasn't very specific with my comment. I see that the current regex uses `\/` to mean `/` (escaping the `/` like you said), which makes sense for Mac and Linux, but...

> does this work on Windows, or does the `/` need to be something silly like `[\\/]`?

I don't remember if webpack uses `/` or `\` here when running on Windows. If it uses `\` then this might not correctly match on Windows, in which case we'll need this regex to match `/` or `\`. If you put the `/` inside of `[]`, I think you won't need to escape it any more... but I believe you _will_ need to escape the `\`, so that leads to `[\\/]`.

Or, if webpack normalizes paths to use `/` only, then what you have is probably fine. I just don't remember for sure. :sweat_smile: